index.js

```diff
@@ -726,34 +752,34 @@
   //.
   //. An array of [types][]:
   //.
-  //.   - [`AnyFunction`][]
-  //.   - [`Arguments`][]
-  //.   - [`Array`][]
-  //.   - [`Boolean`][]
-  //.   - [`Date`][]
-  //.   - [`Error`][]
-  //.   - [`Null`][]
-  //.   - [`Number`][]
-  //.   - [`Object`][]
-  //.   - [`RegExp`][]
-  //.   - [`StrMap`][]
-  //.   - [`String`][]
-  //.   - [`Undefined`][]
-  var env = applyParameterizedTypes([
+  //.   - <code><a href="#AnyFunction">AnyFunction</a></code>
+  //.   - <code><a href="#Arguments">Arguments</a></code>
+  //.   - <code><a href="#Array">Array</a>(<a href="#Unknown">Unknown</a>)</code>
+  //.   - <code><a href="#Boolean">Boolean</a></code>
+  //.   - <code><a href="#Date">Date</a></code>
+  //.   - <code><a href="#Error">Error</a></code>
+  //.   - <code><a href="#Null">Null</a></code>
+  //.   - <code><a href="#Number">Number</a></code>
+  //.   - <code><a href="#Object">Object</a></code>
+  //.   - <code><a href="#RegExp">RegExp</a></code>
+  //.   - <code><a href="#StrMap">StrMap</a>(<a href="#Unknown">Unknown</a>)</code>
+  //.   - <code><a href="#String">String</a></code>
+  //.   - <code><a href="#Undefined">Undefined</a></code>
```

Comment:
I want to generate…

  - <code><a href="https://github.com/sanctuary-js/sanctuary-def#StrMap">StrMap</a>(<a href="https://github.com/sanctuary-js/sanctuary-def#Unknown">Unknown</a>)</code>

rather than…

  - [`StrMap`][1]`(`[`Unknown`][2]`)`

and the only way to have two links inside a `<code>` element is to use HTML.


[1]: https://github.com/sanctuary-js/sanctuary-def#StrMap
[2]: https://github.com/sanctuary-js/sanctuary-def#Unknown